articles/azure-app-configuration/concept-private-endpoint.md

```diff
@@ -61,6 +61,20 @@ If you are using a custom DNS server on your network, you need to configure it t
 
 Enabling private endpoints requires a [Standard or Premium tier](https://azure.microsoft.com/pricing/details/app-configuration/) App Configuration store. To learn about private link pricing details, see [Azure Private Link pricing](https://azure.microsoft.com/pricing/details/private-link).
 
+## Troubleshooting private endpoint errors
+
+### Troubleshoot RP registration errors
+
+The following error indicates that the private endpoint being connected to an App Configuration store is in a subscription that has not registered the Azure App Configuration resource provider:
+
+> The private endpoint's subscription '00000000-0000-0000-0000-000000000000' is not registered to use resource provider 'Microsoft.AppConfiguration'. See https://aka.ms/appconfig/PrivateEndpointTroubleshooting for instructions on how to register a subscription to a resource provider.
```

Comment:
Suggestions:
* Replace this fake GUID with the following one from our list of approved GUIDs:
* Use a site-relative link instead of an aka.ms link. For example (./concept-private-endpoint.md#troubleshooting-private-endpoint-errors)
* This link sends customers to the section they're already reading. If this is where you want to send customers, I suggest removing the sentence, otherwise update the link to another section or article.

```suggestion
> The private endpoint's subscription 'aaaa0a0a-bb1b-cc2c-dd3d-eeeeee4e4e4e' is not registered to use resource provider 'Microsoft.AppConfiguration'.
```

Reply:
Thanks for the suggestions, all have been addressed.